packages/block-library/src/image/index.js

```diff
@@ -231,7 +269,16 @@ export const settings = {
 
 		const figure = (
 			<Fragment>
-				{ href ? <a href={ href } target={ linkTarget } rel={ linkTarget === '_blank' ? 'noreferrer noopener' : undefined }>{ image }</a> : image }
+				{ href ? (
+					<a
+						className={ linkClass }
+						href={ href }
+						target={ linkTarget }
+						rel={ rel }
+					>
+						{ image }
+					</a>
```

Comment:
I wonder if this change creates an "invalid image" warning if you try to upgrade from an image with a link set to open a new tab.

 * checkout master
 * add an image with an external link
 * save
 * checkout this branch
 * refresh

The image shouldn't show up as invalid. I wonder if we should add a deprecated version for this?

To clarify: I didn't have the time to test it but it's a guess.

Reply:
I took this for a spin, and things look like they are working as intended.

Screenshot from master:

<img width="952" alt="screenshot 2018-11-19 at 11 32 48" src="https://user-images.githubusercontent.com/1204802/48701516-fb8f0c80-ebee-11e8-9b5e-e043125b173f.png">

Checked out this branch and reloaded:

<img width="952" alt="screenshot 2018-11-19 at 11 33 05" src="https://user-images.githubusercontent.com/1204802/48701520-0053c080-ebef-11e8-888b-c0525e67b819.png">

I suspect the problem only occurs when you go the other way. Which is adding the attributes, then checking out a branch that removes them. And sure enough:

<img width="949" alt="screenshot 2018-11-19 at 11 34 52" src="https://user-images.githubusercontent.com/1204802/48701623-37c26d00-ebef-11e8-8300-d0d7eca3b840.png">

Going back to master:

<img width="948" alt="screenshot 2018-11-19 at 11 35 16" src="https://user-images.githubusercontent.com/1204802/48701626-38f39a00-ebef-11e8-9fc6-d98fb0d51f07.png">

But this shouldn't be a problem unless we choose to remove this feature :D

Reply:
Checked again, this time I checkout master first, and set the open externally and added a custom link:

<img width="952" alt="screenshot 2018-11-19 at 11 41 03" src="https://user-images.githubusercontent.com/1204802/48702037-275ec200-ebf0-11e8-8079-f35b08d40bd2.png">

Checking out this branch and reloading still worked as expected:

<img width="954" alt="screenshot 2018-11-19 at 11 41 35" src="https://user-images.githubusercontent.com/1204802/48702050-2fb6fd00-ebf0-11e8-8dbe-4c3d0e0a2b94.png">

👍 👍

Reply:
Thanks for the tests @jasmussen :)